app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,21 @@
     <string name="app_name">Accrescent</string>
     <string name="app_list">Apps</string>
     <string name="app_updates">Updates</string>
+    <string name="test">Just kidding. This is a shim.</string>
+    <string name="install">Install</string>
+    <string name="swipe_refresh">Swipe down to refresh</string>
+    <string name="user_action_required">User action required</string>
+    <string name="cant_find_app">Sorry, we couldn\'t find that app</string>
+    <string name="version">Version: %1$s</string>
+    <string name="version_code">Version code: %1$d</string>
+    <string name="device_language_unsupported">Your device\'s language is not supported</string>
+    <string name="device_density_unsupported">Your device\'s screen density is not supported</string>
+    <string name="device_abi_unsupported">Your device\'s ABI/architecture is not supported</string>
```

Comment:
nit: Say either "ABIs are" or "architecture is" instead of both. Singular "ABI" isn't always technically correct since devices can support multiple ABIs.